docker/Dockerfile.vineyardd

```diff
@@ -73,17 +75,23 @@ RUN cd /work/v6d && \
     make -j`nproc` && \
     strip ./bin/vineyardd
 
-# debug has busybox
-FROM gcr.io/distroless/base:debug
+FROM pingcap/alpine-glibc:alpine-3.14.6 as release
```

Comment:
Use the `distroless/base:debug` image is by design for glibc dependencies.

Alpine based images won't work for certain syscalls.